espRFID/UpdateACLTask.cpp

```diff
@@ -96,6 +106,12 @@ void UpdateACLTask::manualUpdate()
     setRunTime(nextAutomatedTime);
     latchTask.blinkDoor();
   }
+  else
+  {
+    incRunTime(nextRetryInterval); 
+    nextAutomatedTime = runTime;
+    retryCount--;
```

Comment:
I don't know if retry count needs to be decremented here.  Manual updates otherwise don't interact with retryCount.